spec/rubocop/cop/performance/count_spec.rb

```diff
@@ -1,54 +1,124 @@
 # frozen_string_literal: true
 
 RSpec.describe RuboCop::Cop::Performance::Count, :config do
-  shared_examples 'selectors' do |selector|
+  shared_examples 'selectors' do |selector, negate_condition|
     it "registers an offense for using array.#{selector}...size" do
       expect_offense(<<~RUBY, selector: selector)
         [1, 2, 3].#{selector} { |e| e.even? }.size
                   ^{selector}^^^^^^^^^^^^^^^^^^^^^ Use `count` instead of `#{selector}...size`.
       RUBY
+
+      if negate_condition
+        expect_correction(<<~RUBY)
+          [1, 2, 3].count { |e| !(e.even?) }
+        RUBY
+      else
+        expect_correction(<<~RUBY)
+          [1, 2, 3].count { |e| e.even? }
+        RUBY
+      end
     end
```

Comment:
I took the liberty of adding correction expectations for all examples which were missing it; let me know if I should move this to a separate PR.